lib/utils.js

```diff
@@ -34,20 +34,18 @@ export const findState = (stateSlug) => {
 
 export const hospitalsInState = (state) => {
   return hospitals.filter(
-    hospital => parametreize(hospital.state) == parametreize(state)
+    (hospital) => parametreize(hospital.state) == parametreize(state)
   );
 };
 
 export const pmusInState = (state) => {
-  return pmus.filter(
-    pmu => parametreize(pmu.state) === parametreize(state)
-  )
+  return pmus.filter((pmu) => parametreize(pmu.state) === parametreize(state));
 };
 
-export const findDonor = (donorSlug) => {
-  return donors[donorSlug];
+export const findDonor = (donorId) => {
+  return donors[donorId];
 };
 
-export const splitText = (text) => {
+export const splitText = (text = "") => {
   return text.trim().split("\n");
 };
```

Comment:
We could safely remove these functions.

Reply:
✅
Resolved via https://github.com/coronasafe/10bedicu/pull/4/commits/24a09668b552a4a9e726c0d214ef329530a7d8d2